helm/ngrok-operator/values.yaml

```diff
@@ -204,6 +204,12 @@ serviceAccount:
   name: ""
   annotations: {}
 
+## Cluster Role Settings
+## @param clusterRole.annotations Additional annotations to add to all ClusterRoles
+##
+clusterRole:
```

Comment:
Since this value gets applied to all the cluster roles, what are your thoughts on making this plural? 
I think a single value to apply to all of them covers 90% of the use cases and is all we are looking to handle right now. But i'm trying to consider also if one day someone wanted to annotate the CRD's separately from the operator's roles, or have specific annotations for 1 CRD but not another (maybe users can make cloud endpoints, but can't delete domains for example). I think we could have this single value apply to all of them and in the future allow specifying `.clusterRole[crdName].annotations` passively